### PR TITLE
Improve the port-forward websocket protocol

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: ws/portforward/model.go [hunk 1]
```diff
@@ -15,14 +15,29 @@
...
+	MessageTypePortError = "error"
 )
 
-const (
-	PropertyPortForwardConnectionID = "connection_id"
-	PropertyPortForwardRemoteHost   = "remote_host"
-	PropertyPortForwardRemotePort   = "remote_port"
-	PropertyPortForwardProtocol     = "protocol"
-)
+// The Error struct is passed in the Body of MsgProto in case the message type is ErrorMessage
```
Comment (alfrunes) [inline ws/portforward/model.go:30]:
```suggestion
// PortForwardNew....
```

File: ws/portforward/model.go [hunk 2]
```diff
@@ -15,14 +15,29 @@
...
+type PortForwardNew struct {
+	RemoteHost *string `msgpack:"remote_host" json:"remote_host"`
+	RemotePort *uint   `msgpack:"remote_port" json:"remote_port"`
+	Protocol   *string `msgpack:"protocol" json:"protocol"`
```
Comment (alfrunes) [inline ws/portforward/model.go:34]: This is `tcp`/`udp` right?

Comment (tranchitella) [reply ws/portforward/model.go:34]: Yes

Comment (tranchitella) [reply ws/portforward/model.go]: I reworked it as a custom type and provided the possible values as constants.

**Review by alfrunes — APPROVED** [review]

**Pull request merged**